xscen/extract.py

```diff
@@ -946,37 +948,45 @@ def get_warming_level(
        [start, end] of the base period. The warming is calculated with respect to it. The default is ["1850", "1900"].
     ignore_member : bool
        Decides whether to ignore the member when searching for the model run in tas_csv.
-    tas_csv : str, optional
-       Path to a csv of annual global mean temperature with a row for each year and a column for each dataset.
-       If None, it will default to data/IPCC_annual_global_tas.csv which was built from
+    tas_src : str
+       Path to a netCDF of annual global mean temperature (tas) with an annual "time" dimension
+       and a "simulation" dimension with the following coordinates: "mip_era", "source", "experiment" and "member".
+       If None, it will default to data/IPCC_annual_global_tas.nc which was built from
        the IPCC atlas data from  Iturbide et al., 2020 (https://doi.org/10.5194/essd-12-2959-2020)
-       and extra data from pilot models of MRCC5 and ClimEx.
+       and extra data for missing CMIP6 models and pilot models of MRCC5 and ClimEx.
     return_horizon: bool
         If True, the output will be a list following the format ['start_yr', 'end_yr']
         If False, the output will be a string representing the middle of the period.
+    output: {'selected', 'requested'}
+        When `realization` is a sequence.
+        If "requested", the function returns a list of the same length as `realization`.
+        If "selected", the function returns a dict, keys are the models selected in the database.
```

Comment:
Because the previous output for a sequence of datasets was a dictionary and dictionaries can't have the same key twice, the output was not always of the same length. Best example is with RCMs : if I have a list of realisations where I have multiple RCMs being driven by the same GCM, that GCM appears only once. Finding which horizon in the output corresponds to which realisation is thus made awkward.

I thought that this expected! If you think that's a bug we can also completely remove the `output` argument and always return the "requested" version.